src/CrossEncoderSearch.py

```diff
@@ -0,0 +1,25 @@
+import os
+import re
+from sentence_transformers import CrossEncoder
+from sentence_transformers import SentenceTransformer
+
+class CrossencoderSearch:
+
+    def __init__(self,query,retrieved_docs,run_cross_encoding=False):
+
+        self.query=query
+        self.pairs=retrieved_docs
+        self.final_pairs=[]
+        if run_cross_encoding:
```

Comment:
Since we are only using this class for cross encoding, lets remove this flag as it is redundant now. We can delete the variable run_cross_encoding from this code

Reply:
So I'll remove it from both config as well as from the search by index python file ?